File: Npc/11000097.cs
using Maple2.Model.Enum;
using Maple2.Script.Npc;

/// <summary>
/// 11000097: Solvay
/// </summary>
public class _11000097 : NpcScript {
    protected override int First() {
        return 20;
    }

    // Select 0:
    // $script:0831180407000381$
    // - What seems to be the problem?
    protected override int Select() => 0;

    protected override int Execute(int selection) {
        switch (Id, Index++) {
            case (20, 0):
                // $script:0831180407000382$
                // - The Barrota Trading Company is Maple World's biggest mercantile organization. Its hawkers go everywhere in the world, no matter how steep or dangerous.
                return -1;
        }

        return default;
    }

    protected override NpcTalkButton Button() {
        return (Id, Index) switch {
            (20, 0) => NpcTalkButton.Close,
            _ => NpcTalkButton.None,
        };
    }
}

File: Npc/11000097.py
""" 11000097: Solvay """
from npc_api import Option, Script
import random


class Main(Script):
    def first(self) -> int:
        return 20

    def select(self) -> int:
        return 0

    def __0(self, pick: int) -> int:
        # $script:0831180407000381$
        # - What seems to be the problem?
        return None # TODO

    def __20(self, pick: int) -> int:
        # $script:0831180407000382$
        # - The Barrota Trading Company is Maple World's biggest mercantile organization. Its hawkers go everywhere in the world, no matter how steep or dangerous.
        return -1

    def button(self) -> Option:
        if (self.state, self.index) == (20, 0):
            return Option.CLOSE
        return Option.NONE
